### DISCUSSION TITLE
How to add additional repositories to a regex Manager

### DISCUSSION THREAD

**Discussion by timguy** [renovatebot/renovate#22767] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate looks up dependencies in company nexus.

### What you tried so far.

This is my config and searchs my ivy.xml and resolves the maven dependencies.
But some dependencies are in the company nexus.
So how to use this nexus repo instead or additional to maven?
```

    "regexManagers": [
    {
      "fileMatch": ["ivy\\.xml$"],
      "matchStrings": [
        "<dependency\\s+org=\"(?<ivyOrg>[^\"]+)\"\\s+name=\"(?<ivyName>[^\"]+)\"\\s+rev=\"(?<currentValue>[^\"]+)\""
      ],
      "datasourceTemplate": "maven",
      "versioningTemplate": "maven",
      "depNameTemplate": "{{ivyOrg}}:{{ivyName}}"
    }
  ],
  
  "hostRules": [
    {
      "matchHost": "https://abc.companytooling.com/nexus/repository/project-grouped/",
      "username": "{{ secrets.ABC_NEXUS_USER }}",
      "password": "{{ secrets.ABC_NEXUS_PASSWORD }}"
    }
  ]
}
```

- Adding the `hostRules `did not helped like in gradle projects
- Including the hosRules into the regexManager section is also not allowed ("Regex Manager contains disallowed fields: hostRules")

**Comment by rarkins** [comment]
https://docs.renovatebot.com/configuration-options/#registryurltemplate
